app/lib/rocketchat/services/restApi.ts

```diff
@@ -57,7 +57,7 @@ export const e2eUpdateGroupKey = (uid: string, rid: string, key: string): any =>
 	// @ts-ignore
 	sdk.post('e2e.updateGroupKey', { uid, rid, key });
 
-export const e2eRequestRoomKey = (rid: string, e2eKeyId: string) =>
+export const e2eRequestRoomKey = (rid: string, e2eKeyId: string): Promise<{ message: { msg?: string }; success: boolean }> =>
```

Comment:
Do we need this `success: boolean` ?

Reply:
As it doesn't go through ResultFor it doesn't add success automatically.
This object is basically what comes from the api.